### ISSUE TITLE
PLC temeperaturesensor don´t work

### ISSUE THREAD

**Issue by juspa1978** [Feilner/homebridge-plc#27]
When tryed to use PLC TemperatureSensor i don´t get accessory to homekit room screen 
"accessory": "PLC_TemperatureSensor",
					"name": "Temperature",
					"db": 9,
					"get_CurrentTemperature": 16
					"enablePolling": true,
					"pollInterval": 60'
If i modify "accessory": "PLC_TemperatureSensor", -> "accessory": "Temperature", then i get message on homeapp that accessory is not currently supported

**Comment by Feilner** [comment]
Can you post your complete config?
Have you added it inside a platform config?
E.g.
{
"platform": "PLC",
"ip": "10.10.10.32",
"rack": 0,
"slot": 2,
"enablePolling": true,
"defaultPollInterval": 15,
"distributePolling": true,
"enablePush": true,
"enableControl": true,
"port": 8888,
"accessories": [
{
Here
}
]

Am So, 31. Jan 2021, um 09:05, schrieb juspa1978:

**Comment by juspa1978** [comment]
"platform": "PLC",
"ip": "192.168.0.146",
"rack": 0,
"slot": 1,
"enablePolling": true,
"defaultPollInterval": 15,
"distributePolling": true,
"enablePush": true,
"enableControl": true,
"port": 8888,
"accessories": [

Exsamble ”plc switch” works fine. Problem only with temperature sensor

**Comment by juspa1978** [comment]
![F6C224BF-8052-4E39-865F-87280E599109](https://user-images.githubusercontent.com/76985784/106381304-70118880-63c0-11eb-9495-47f6801aab0f.jpeg)
”Ei tuettu” means = ”homeapp not currently supporting this device”

**Comment by Feilner** [comment]
The picture from the home App?
The logo does not fit. I have never saw this. 
It should look like the attached. 
Are you using homebridge-UI? Can you have a look how the accessory is displayed there?
Did you head a look at the log wir enabled debug mode?


Am So, 31. Jan 2021, um 11:35, schrieb juspa1978:

**Comment by juspa1978** [comment]
Yes pic is from apple homekitapp (from phone)
At hoobs accessories page i can see temperature correctly.

**Comment by Feilner** [comment]
Ok now we get more light in the mystery.
You are not using homebridge you are using hoobs.

I have short downloaded and tested hoobs. For this I used the hoobs docker container.
I can confirm that the plc plugin works also works with hoobs and all accessories are correctly shown at the hoobs accessory page.

I also connected hoobs to the home app and all my accessories including the PLC_TemperatureSensor are working fine.

// to test it I executed the following on a ubuntu machine (I added no storage so it had no history at restert)
docker run --net=host hoobs/hoobs

Best regards,

Oliver

 


Am So, 31. Jan 2021, um 12:20, schrieb juspa1978:
